com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -475,6 +475,8 @@ static void RenderCameraStack(ScriptableRenderContext context, Camera baseCamera
 
             bool isStackedRendering = lastActiveOverlayCameraIndex != -1;
 
+            // Update volumeframework before initializing additional camera data
+            UpdateVolumeFramework(baseCamera, baseCameraAdditionalData);
```

Comment:
There are other calls to UpdateVolumeFramework happening after InitializeCameraData, shouldn't they be changed as well?
 
 What's in the InitializeCameraData that depends on the UpdateVolumeFramework?

Reply:
Hey felipe. It's the [CheckPostProcessForDepth](https://github.com/Unity-Technologies/Graphics/blob/c6038de9fbc56cbc73ca43d112e79dd0bac725c3/com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs#L620) called in [InitializeAdditionalCameraData](https://github.com/Unity-Technologies/Graphics/blob/c6038de9fbc56cbc73ca43d112e79dd0bac725c3/com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs#L831
) that depends on the UpdateVolumeFramework. 
UpdateVolumeFramework will update volume manager with the current camera. And that need to happen before the CheckPostProcessForDepth call.

Reply:
Makes sense now. it also seems to make sense that we should guarantee that all calls to UpdateVolumeFramework should happen before InitializeAdditionalCameraData, Otherwise we might get other issues in the future?

Reply:
I agree. I did a search in Graphics repo looking for UpdateVolumeFramework use cases. There are 3 places that it is called. I verified with the PR change, all 3 cases have UpdateVolumeFramework  called before InitializeAdditionalCameraData. I think we are good.

Reply:
Hey @phi-lira, could you review my last comment about this PR and give some feedback :)? Thanks!